### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by aimichal**
This explains how to configure Docker to use this helper for specific registries.

### CODE REVIEW

**Comment by samuelkarp** [comment]
@nrdlngr Can you take a look?

**Comment by aimichal** [comment]
Ping?

File: README.md [hunk 1]
```diff
@@ -49,6 +49,20 @@ of your `~/.docker/config.json` file to be:
 }
 ```
 
+This will cause `docker` to use `docker-credential-ecr-login` for all remote registries.
+
+To use this helper for a specific ECR registry, use a `credHelpers` section with the name of your ECR registry:
```
Comment (nrdlngr) [inline README.md]: To use this credential helper for a specific ECR registry, create a `credHelpers` section with the URI of your ECR registry:

File: README.md [hunk 2]
```diff
@@ -49,6 +49,20 @@ of your `~/.docker/config.json` file to be:
...
+
+```json
+{
+	"credHelpers": {
+		"aws_account_id.dkr.ecr.region.amazonaws.com": "ecr-login"
+	}
+}
+```
+
+This is useful if you use `docker` to operate on registries that have different authentication credentials.
```
Comment (nrdlngr) [inline README.md]: This is useful if you use `docker` to operate on registries that use different authentication credentials.

**Comment by aimichal** [comment]
Thanks for the suggestions. Wording updated.

**Comment by samuelkarp** [comment]
@aimichal Thanks for the contribution!

**Pull request merged**